File: src/extension/compass/__tests__/fixtures/sample_gaps.java
package demo;

class GapService {
    void run() {
        GapWidget widget = new GapWidget();
        Box<GapWidget> box = new Box<GapWidget>();
        widget.render();
        Registry.lookup(1);
        localHelper();
    }

    void localHelper() {
    }
}

class GapWidget {
    void render() {
    }
}

File: src/extension/compass/__tests__/fixtures/sample_gaps.cs
namespace Demo
{
    public class GapService
    {
        public void Run()
        {
            var widget = new GapWidget();
            var box = new Box<int>();
            var nested = new Outer.Inner();
            widget.Render();
            Registry.Lookup(1);
            LocalHelper();
        }

        private void LocalHelper()
        {
        }
    }

    public class GapWidget
    {
        public void Render()
        {
        }
    }
}
